### PR TITLE
[DO NOT MERGE] [ACR] ARM SystemData for Resource definition

### PR DESCRIPTION
**Pull request by hkuadithya**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

@djyou @ankurkhemani @jaysterp 
Could you please review the changes?

### CODE REVIEW

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-12-01-preview/containerregistry.json [hunk 1]
```diff
@@ -2730,6 +2730,57 @@
...
+            "modelAsString": true
+          }
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "description": "The timestamp of resource last modification (UTC).",
+          "type": "string"
+        }
+      }
+    },
```
Comment (akning-ms) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-12-01-preview/containerregistry.json:2783]: Can we reuse definition in specification\common-types\resource-management\v1\types.json for SystemData?

Comment (hkuadithya) [reply specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-12-01-preview/containerregistry.json]: @akning-ms done. We're reusing the SystemData definition from `common-types`.

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json [hunk 2]
```diff
@@ -367,21 +367,6 @@
         "x-ms-examples": {
           "Registries_ScheduleRun": {
             "$ref": "./examples/RegistriesScheduleRun.json"
-          },
```
Comment (northtyphoon) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json:370]: Do you need to change?

**Comment by ravbhatnagar** [comment]
Removing ARM Feedback label since ARM review board will not be reviewing property level changes going forward.

**Pull request closed without merging**